packages/compass-components/src/components/interactive-popover.tsx

```diff
@@ -50,6 +50,8 @@ type InteractivePopoverProps = {
     ref: React.LegacyRef<HTMLButtonElement>;
     children: React.ReactNode;
   }) => React.ReactElement;
+  hasCustomCloseButton?: boolean;
+  customTrapFallback?: string;
```

Comment:
Can we use a better name here? It's custom because you are already providing a prop. What it's supposed to trap?

Reply:
`customFocusTrapFallback` at least so we know this is not related to the existing `containedElements` prop that also affects trap boundaries, but for close on click and not for focus?

Reply:
Yeah, that sounds good, if it's more explicit and it helps is just changing one name.